b2b-sample/guardio-insurance-sample-application/components/data/templates/facebook.json

```diff
@@ -104,47 +104,10 @@
           "authenticatorId": "RmFjZWJvb2tBdXRoZW50aWNhdG9y",
           "isEnabled": true,
           "properties": [
-            {
-              "key": "AuthnEndpoint",
-              "type": "STRING",
-              "displayOrder": 0,
-              "regex": ".*",
-              "isMandatory": false,
-              "isConfidential": false,
-              "options": [],
-              "defaultValue": "",
-              "subProperties": []
-            },
-            {
-              "key": "Scope",
-              "displayName": "Scope",
-              "description": "Enter a comma separated list of permissions to request from the user",
-              "type": "STRING",
-              "displayOrder": 3,
-              "regex": ".*",
-              "isMandatory": false,
-              "isConfidential": false,
-              "options": [],
-              "defaultValue": "email",
-              "subProperties": []
-            },
-            {
-              "key": "UserInfoFields",
-              "displayName": "User Information Fields",
-              "description": "Enter comma-separated user information fields you want to retrieve",
-              "type": "STRING",
-              "displayOrder": 4,
-              "regex": ".*",
-              "isMandatory": false,
-              "isConfidential": false,
-              "options": [],
-              "defaultValue": "",
-              "subProperties": []
-            },
             {
               "key": "ClientId",
-              "displayName": "Client Id",
-              "description": "Enter Facebook client identifier value",
+              "displayName": "App ID",
+              "description": "The generated unique ID which is generated when the Facebook OAuth app is created.",
```

Comment:
Suggestion
`The unique ID which is generated when the Facebook OAuth app is created.`

Reply:
The `facebook.json` file has been removed with a later PR,
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-admin-app/data-access/data-access-common-models-util/src/lib/identityProvider/data/templates